### ISSUE TITLE
getFirst: command not found

### ISSUE THREAD

**Issue by ghost** [CharlesSchimmel/nowPlaying#3]
When I run mpd-notify start, this is whats outputted: 

```
 *Starting mpd-notify.
/usr/local/bin/mpd-notify: line 115: getFirst: command not found
/usr/local/bin/mpd-notify: line 130: getFirst: command not found
```
I'm pretty sure this isn't supposed to happen, any idea why?

**Comment by CharlesSchimmel** [comment]
I'm guessing your environment is different than mine. What are you running?

**Comment by CharlesSchimmel** [comment]
On second thought, I think that's not the issue. I'll look into it and get back to you. Thanks for checking out this project though, how did you find it?

**Comment by dinojr** [comment]
I'm having the same problem, is getFirst supposed to be a shell command ?

**Comment by CharlesSchimmel** [comment]
No, it's a function. I've pushed an update that seems to work for me, let me know if it works for you. @tohd @dinojr 

It's been a while since I worked on this and honestly the code isn't great. I've been in the process of moving the project to Haskell: https://github.com/CharlesSchimmel/mpd-notify-hs which should make it easier to maintain as well as run lighter. What features of this project appeal to you? I'll try to include them in the Haskell version.

**Comment by dinojr** [comment]
No more errors for me thanks to your update. However I couldn't check if it works as expected since I'm only interested in the wallpaper setting capability and for some reasons feh -bg-max doesn't seem to work on my system (gnome-shell).
